### ISSUE TITLE
CropCircleTransformation is Deprecated? What now?

### ISSUE THREAD

**Issue by jacobceles** [wasabeef/glide-transformations#103]
I'm using the 3.0.1 version. When I try to use CropCircleTransformation it says  _'jp.wasabeef.glide.transformations.CropCircleTransformation' is deprecated_. So then what's the new one? I didn't see any alternatives in the README.

**Comment by CrabeMan** [comment]
As written [on the top of the class definition,](https://github.com/wasabeef/glide-transformations/blob/master/transformations/src/main/java/jp/wasabeef/glide/transformations/CropCircleTransformation.java#L27) you have to use 
```
options.apply(RequestOptions.circleCropTransform())
```

CrabeMan

**Comment by jacobceles** [comment]
But isn't that part of glide itslef? I mean I figured it out and implemented it without compiling this library. So my question is more like, what's for this library?

**Comment by CrabeMan** [comment]
Yes with Glide4 they have implemented some defaults transformations as [`CircleCrop`](https://github.com/bumptech/glide/blob/master/library/src/main/java/com/bumptech/glide/load/resource/bitmap/CircleCrop.java) or [`RoundedCorners`](https://github.com/bumptech/glide/blob/master/library/src/main/java/com/bumptech/glide/load/resource/bitmap/RoundedCorners.java).
Check at [`TransformationUtils`](https://github.com/bumptech/glide/blob/master/library/src/main/java/com/bumptech/glide/load/resource/bitmap/TransformationUtils.java).
If you don't need more transformation than these, you don't need to use this library.

CrabeMan.

**Comment by Ashish19901** [comment]
used
 implementation 'jp.wasabeef:glide-transformations:3.3.0'
